README.md

```diff
@@ -350,6 +350,85 @@ Type: `string`
 
 Default: `""`
 
+### <a name="input_subscription_register_resource_providers"></a> [subscription\_register\_resource\_providers](#input\_subscription\_register\_resource\_providers)
+
+Description: The list of resource providers to register. The defaults are taken from the AzureRM provider.  
```

Comment:
Can we provide a link to the RM provider list so its easy to track and update in the future